ui/packages/consul-ui/app/templates/debug.hbs

```diff
@@ -1,6 +1,16 @@
 {{page-title 'Engineering Docs - Consul' separator=' - '}}
 
 {{document-attrs class="is-debug"}}
+{{! Tell CSS what we have enabled }}
+{{#if (can "use acls")}}
+  {{document-attrs class="has-acls"}}
+{{/if}}
+{{#if (can "use nspaces")}}
+  {{document-attrs class="has-nspaces"}}
+{{/if}}
+{{#if (can "use partitions")}}
+  {{document-attrs class="has-partitions"}}
+{{/if}}
```

Comment:
These just add some root/html classes for our docs when certain things are enabled. It's just copied over from our prod `application.hbs` file where we've always done this, noticed we didn't do it on our docs pages but we should.